src/modules/managedata.js

```diff
@@ -0,0 +1,60 @@
+class Todo {
+  constructor() {
+    this.todos = [];
+  }
+
+  addTodo(description) {
+    this.todos.push({
+      index: this.todos.length + 1,
+      completed: false,
+      description,
+    });
+  }
+
+  removeTodo(index) {
+    this.todos = this.todos.filter((singleTodo) => singleTodo.index !== index);
+    const helpertodos = [];
+    let helpertodo;
+    this.todos.forEach((singleTodo) => {
+      helpertodo = {
+        index: helpertodos.length + 1,
+        completed: singleTodo.completed,
+        description: singleTodo.description,
+      };
+      helpertodos.push(helpertodo);
+    });
+    this.todos = helpertodos;
+  }
+
+  editTodo(index, description) {
+    this.todos = this.todos.map((singleTodo) => {
+      if (singleTodo.index === index) {
+        singleTodo.description = description;
+      }
+      return singleTodo;
+    });
+  }
```

Comment:
- [ ] please make sure when the task is edited is saved at the moment when I edit a task is not saved, please try to fix that